src/transformers/generation_utils.py

```diff
@@ -526,6 +529,18 @@ def _generate_no_beam_search(
         sent_lengths = input_ids.new(batch_size).fill_(max_length)
 
         past = None
+        past2 = None
+
+        # Initialize MMI antiLM settings
+        if antilm_params is None:
+            print("Must define antilm_params")
+            quit()
+        threshold_g = antilm_params["length_thresh"] + cur_len  # cur_len starts at the length of input
+        alpha = antilm_params["alpha"]
+        lm_model = antilm_params["lm_model"]
+        bsz = input_ids.size(0)
+        input_ids2 = torch.LongTensor(bsz,1).fill_(50256).to(input_ids.device)
```

Comment:
is `50256` some symbolic token like `pad_token_id`?

Reply:
Yes, sorry that was hard-coded for GPT-2.